en/04_Changelogs/5.0.0.md

```diff
@@ -112,6 +112,7 @@ This is a major release and as a result there are a number of breaking API chang
 
 ### General changes {#api-general}
 
+- [DataList::sort()](api:SilverStripe\ORM\DataList) No longer accepts raw SQL. A new method `DataList::orderBy()` has been created which accepts raw SQL, though it's recommended to continue using `DataList::sort()` if possible to reduce the chance of SQL injection vulnerabilities.
```

Comment:
This should be in the ORM section below.

Reply:
Updated